src/lib.rs

```diff
@@ -1,8 +1,8 @@
 //! This crate contains the officially supported MongoDB Rust driver, a
 //! client side library that can be used to interact with MongoDB deployments
-//! in Rust applications. It uses the [`bson`](docs.rs/bson) crate for BSON support.
```

Comment:
Links aren't required for external crates specified in this format. `docs.rs/bson` was being treated as a relative link (i.e. `https://docs.rs/mongodb/latest/mongodb/docs.rs/bson`); omitting it fixes the issue and properly links to the bson crate.